os/fs/smartfs/README.txt

```diff
@@ -1,17 +1,16 @@
-SMARTFS README
-^^^^^^^^^^^^^^
+jSmartFs README
+^^^^^^^^^^^^^^^
 
 This README file contains information about the implemenation of the TinyAra
-Sector Mapped Allocation for Really Tiny (SMART) FLASH file system, SMARTFS.
-
+Journaling Sector Mapped Allocation for Really Tiny (jSmart) FLASH file system, jSmartFs.
```

Comment:
Let's write this as "Journaling - Sector Mapped Allocation for Really Tiny Flash File system, jSmartFs"